plugins/bundle/plugin.go

```diff
@@ -116,6 +116,9 @@ func (p *Plugin) Start(ctx context.Context) error {
 	}
 
 	p.loadAndActivateBundlesFromDisk(ctx)
+	if p.BundleLocationLocal() {
```

Comment:
question: how would this handle configuration of a local tar.gz and a remote one at the same time?

Reply:
@charlieegan3 in -s option , asBundle flag is false.

should user be able to load multiple bundles like one from local and another from remote ? like in following:
opa run -s bundle.tar.gz  http://remote.com/bundle.tar.gz

if above case is true, I can try removing the check in my code what you have pointed out.

Reply:
I think that it should be possible to use all the different methods at the same time. e.g.

```
$ tree -a .
├── bundle
│   ├── .manifest
│   └── pkg1.rego
├── bundle2
│   ├── .manifest
│   └── pkg2.rego
├── bundle3
│   ├── .manifest
│   └── pkg3.rego
├── bundle4
│   ├── .manifest
│   └── pkg4.rego
└── config.yaml

4 directories, 8 files

$ find . -type f | grep .rego | sort | xargs -n 1 cat
package pkgs.pkg1

name := "pkg1"
package pkgs.pkg2

name := "pkg2"
package pkgs.pkg3

name := "pkg3"
package pkgs.pkg4

name := "pkg4"

$ find . -type f | grep manifest | sort | xargs -n 1 cat
{
  "roots": ["pkgs/pkg1"]
}
{
  "roots": ["pkgs/pkg2"]
}
{
  "roots": ["pkgs/pkg3"]
}
{
  "roots": ["pkgs/pkg4"]
}

$ opa build -b bundle2 -o bundle2.tar.gz
$ opa build -b bundle3 -o bundle3.tar.gz
$ opa build -b bundle4 -o bundle4.tar.gz

# serve the local dir over http
$ ruby -run -ehttpd . -p8000

$ cat config.yaml 
services:
  local:
    url: http://localhost:8000

bundles:
  authz:
    service: local
    resource: bundle3.tar.gz

# load some bundles from config, some from cmd line, some over http from cmd line
$ opa run -s --config-file=config.yaml -b bundle2.tar.gz -b bundle http://localhost:8000/bundle4.tar.gz

$ curl --silent "http://localhost:8181/v1/data/pkgs" | jq
{
  "result": {
    "pkg1": {
      "name": "pkg1"
    },
    "pkg2": {
      "name": "pkg2"
    },
    "pkg3": {
      "name": "pkg3"
    },
    "pkg4": {
      "name": "pkg4"
    }
  }
}
```